lib-gphotos/client.go

```diff
@@ -145,17 +215,20 @@ func (client *Client) UploadFile(filePath string, pAlbumID ...string) (*photosli
 		result := batchResponse.NewMediaItemResults[0]
 		if result.Status.Message != "OK" && result.Status.Message != "Success" {
 			// TODO: We should use a different field like `googleapi.ServerResponse`
-			return nil, stacktrace.NewError("status message should be OK/Succecc, found: %s", result.Status.Message)
+			return nil, stacktrace.NewError("status message should be OK/Success, found: %s", result.Status.Message)
 		}
 
-		log.Printf("%s uploaded successfully as %s", filename, result.MediaItem.Id)
+		// Clear uploadURL as upload was completed
+		uploadURL = new(string)
```

Comment:
Why do we clear `uploadURL` before returning?

Reply:
To let the client of the google-photos-api-client-go API remove the upload URL from its database.

Reply:
https://github.com/nmrshll/gphotos-uploader-cli/pull/63/files#diff-62ec878e0d5cfa576428c0c6862adbcaR96

Reply:
After rechecking, it avoids letting the client believe the upload URL may need to be stored for later reuse, while it is in the `final` state (i.e. not `active`) and cannot used anymore.